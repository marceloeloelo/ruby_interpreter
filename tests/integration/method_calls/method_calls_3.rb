puts ("hola".length())

b = []

puts (b.length())

a = 'd'

puts ("hola #{a}".length())

a = [1,'chau']

puts (a[1].length())

puts (a.length())

puts (a[0].length())
