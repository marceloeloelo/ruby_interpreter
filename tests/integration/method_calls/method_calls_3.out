4
0
6
4
2
NoMethodError: undefined method `length' for 1:Fixnum
